test/test_ops.py

```diff
@@ -68,7 +68,8 @@ class TestGradients(TestCase):
     def _get_safe_inplace(self, inplace_variant):
         @wraps(inplace_variant)
         def _fn(t, *args, **kwargs):
-            return inplace_variant(t.clone(), *args, **kwargs)
+            inputs = t.clone() if isinstance(t, torch.Tensor) else [arg.clone() for arg in t]
```

Comment:
nit: is it important that inputs is a list instead of a tuple here?

Reply:
Nope, I made it a tuple. Thanks!